pkg/consts/consts.go

```diff
@@ -442,47 +442,6 @@ const RateLimited = "rate limited"
 // CreatedByTag tag key for CSI drivers
 const CreatedByTag = "k8s-azure-created-by"
 
-// port specific
-const (
```

Comment:
Could we keep them in the consts package (as they are shared by both provider and tests)?

Reply:
the change is reverted.